frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -738,6 +738,28 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_initialize(now: T::BlockNumber) -> Weight {
+			if Self::force_phase().is_some() {
+				match Self::force_phase() {
+					Some(Phase::Signed) => {
+						// If the current phase is not `Phase::Off` proceed to the next round.
+						if !Self::current_phase().is_off() {
+							Self::do_force_rotate_round();
```

Comment:
What happens if the phase is already signed? Should we still rotate round?

Reply:
It makes sense to me to rotate the round. @kianenigma Thoughts?